script/playbackstrategy/modifiers/live/restartable.js

```diff
@@ -6,49 +6,25 @@ define(
   ],
     function (Html5Player, MediaPlayerBase) {
       'use strict';
-      var AUTO_RESUME_WINDOW_START_CUSHION_MILLISECONDS = 8000;
+      var AUTO_RESUME_WINDOW_START_CUSHION_SECONDS = 8;
 
-      function RestartableLivePlayer (deviceConfig, logger) {
-        var mediaPlayer = Html5Player(logger);
-        var millisecondsUntilStartOfWindow;
-        var bufferingStarted;
+      function RestartableLivePlayer (deviceConfig, logger, windowType, timeData) {
```

Comment:
Could always pass in a mediaPlayer instance into this instead of instantiating it... Also stops the implementation being tied to HTML5Player

Reply:
All live modifiers now require the media player passed in.